eng/common/spelling/Invoke-Cspell.ps1

```diff
@@ -170,10 +140,11 @@ try {
   npm ci | Write-Host
 
   # Use the mutated configuration file when calling cspell
-  $command = "npx cspell $JobType --config $CSpellConfigPath --no-must-find-files --root $SpellCheckRoot --relative"
+  $command = "npm exec --no -- cspell $JobType --config $CSpellConfigPath --no-must-find-files --root $SpellCheckRoot --relative"
   Write-Host $command
-  $cspellOutput = npx  `
-    --no-install `
+  $cspellOutput = npm exec  `
```

Comment:
Any reason to not just do `$cspellOutput = & $command`?

Reply:
The potential for injection is the biggest reason that comes to mind... though in this context that potential seems pretty low.